CONTRIBUTING.rst

```diff
@@ -0,0 +1,35 @@
+Contributing to Renku
+=====================
+
+Want to contribute to Renku? Thanks! There are many ways to help, and we very much
+appreciate your efforts.
+
+The sections below provide guidelines for various types of contributions.
+
+Bug Reports and Feature Requests
+===========
```

Comment:
The underline has to be the length of the text above otherwise Sphinx complains

Reply:
Fixed.